Dockerfile

```diff
@@ -63,29 +62,40 @@ RUN HDF5_INCDIR=/usr/include/hdf5/serial
 RUN pip install --upgrade pip 
 RUN pip install -r requirements.txt
 
-# Install GDAL 2.4.1 from source
-RUN wget http://download.osgeo.org/gdal/2.4.1/gdal-2.4.1.tar.gz \
-    && tar -xzf gdal-2.4.1.tar.gz \
-    && rm gdal-2.4.1.tar.gz
+# TODO: not sure that we need this?
```

Comment:
Yeah, we use gdal for metadata extraction for rasters and features

Reply:
thanks @sblack-usu I should've been more specific. I know we need gdal.
But we install gdal-bin above with apt... so I wasn't sure that we needed to wget it and install again...but maybe we need to in order to get GEOS support?

Reply:
ah, I didn't realize it got installed with apt already.  Maybe you can remove it 🤷

Reply:
attempting with removal and we shall see...